src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! A Rust implementation of the [GeoArrow](https://github.com/geoarrow/geoarrow) specification,
 //! plus algorithms implemented on and returning these GeoArrow arrays.
-
+#![feature(iter_advance_by)]
```

Comment:
I don't want to use anything outside of standard rust as something required in this library;

```
: `#![feature]` may not be used on the stable release channel
```

Reply:
sure, I don't want to use it too.